centinel/views.py

```diff
@@ -15,9 +17,9 @@
 from werkzeug import secure_filename
 
 
-from centinel.models import Client, Role
-import config
+from centinel.as_info import ASInfo
```

Comment:
Hmm. I don't love underscores in filenames, but if that is our existing convention we can leave it.